backend/src/main/modules/helper.py

```diff
@@ -0,0 +1,1010 @@
+        payload = json.dumps({
+            "externalId": externalId,
+            "name": pName,
+            "description": "Program",
+            "resourceType": [
+                "program"
+            ],
+            "language": [
+                "English"
+            ],
+            "keywords": [],
+            "concepts": [],
+            "createdFor": orgIds,
+            "rootOrganisations": orgIds,
+            "startDate": startDateOfProgram,
+            "endDate": endDateOfProgram,
+            "imageCompression": {
+                "quality": 10
+            },
+            "creator": creatorName,
+            "owner": creatorKeyCloakId,
+            "author": creatorKeyCloakId,
+            # "metaInformation": {
+            #     # "state":entitiesPGM.split(","),
+            #     # "role": mainRole.split(",")
+            #     },
+                "requestForPIIConsent":False
+            })
+
+        
+        headers = {'X-authenticated-user-token': accessToken,
+                   'internal-access-token': internal_access_token,
+                   'Content-Type': 'application/json',
+                   'Authorization':authorization}
+
+        # program creation 
+        responsePgmCreate = requests.request("POST", programCreationurl, headers=headers, data=(payload))
+        print(responsePgmCreate.text)
+
+        fileheader = [pName, ('Program Sheet Validation'), ('Passed')]
+        if responsePgmCreate.status_code == 200:
+            responsePgmCreateResp = responsePgmCreate.json()
+            programIdForSuccess = responsePgmCreateResp['result']["_id"]
+        else:
+            # terminate execution
+            print("Program creation API failed. Please check logs.")
+
+# Th    is function checks for the sequince
+
+    
+
+    def createFileStructForProgram(programFile):
+        print("programFile:-------------",programFile)
+        if not os.path.isdir('resourceFile'):
+            os.mkdir('resourceFile')
+        if "/" in str(programFile):
+            fileNameSplit = str(programFile).split('/')[-1:]
+            print("newfileNameSplit ;",fileNameSplit)
+        else :
+
+            fileNameSplit = os.path.basename(programFile)
+        print("updatedfileNameSplit : ",fileNameSplit)
+
+        if isinstance(fileNameSplit, list):
+            fileNameSplit = fileNameSplit[0]
+        # fileNameSplit = str(programFile)
+        if fileNameSplit.endswith(".xlsx"):
+            print("latest fileNameSplit",fileNameSplit)
+            ts = str(time.time()).replace(".", "_")
+            
+            folderName = fileNameSplit.replace(".xlsx", "-" + str(ts))
+            print("folderName :",folderName)
+            os.mkdir('resourceFile/' + str(folderName))
+            path = os.path.join('resourceFile', str(folderName))
+            print("done",path)
+        else:
+            print("something")
+        returnPathStr = os.path.join('resourceFile', str(folderName))
```

Comment:
is this code out side else ? 
If else don't have any code , why should we keep it?